### ISSUE TITLE
Move OpenTelemetry Jaeger to Quarkiverse

### ISSUE THREAD

**Issue by kenfinnigan** [quarkusio/quarkus#17771]
## Description
With the addition of the OTLP Exporter for OpenTelemetry, we should move the Jaeger Exporter to a Quarkiverse extension as we want to focus on the OTLP Exporter

**Comment by aloubyansky** [comment]
@kenfinnigan will you be the/a maintainer?

**Comment by aloubyansky** [comment]
And what should the `groupId` and `artifactId` be?

**Comment by kenfinnigan** [comment]
Possibly, the other option is we just deprecate it and then remove.

Haven't thought about Ids for either yet.

@ebullient, what do you think? Keep the Jaeger exporter in Quarkiverse or just deprecated in favor of OTLP?

**Comment by ebullient** [comment]
I would check on usage. If that exporter is getting reasonable use, a move to quarkiverse would allow people to keep using it (as it was).. moving to quarkiverse also means less stuff in the main repo (which is a side benefit), though it does bring some risk here and there for testing (intersection/usage of micrometer)

**Comment by rsvoboda** [comment]
`OpenTelemetry exporter: Jaeger` is marked as TECH-PREVIEW for RHBQ 2.2

https://issues.redhat.com/browse/QUARKUS-280?focusedCommentId=18934906&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-18934906 -- We agreed that the OpenTelemetry and OpenTelemetry Jaeger extensions would be "Tech Preview" for 2.2

**Comment by kenfinnigan** [comment]
Right, but the Jaeger extension won't go beyond "Tech Preview", it will be replaced with the OTLP one in product.

Once the OTLP extension is in product, the Jaegar one would be removed

**Comment by rsvoboda** [comment]
OKi, by that time I think groupId and artifactId of OpenTelemetry Jager shouldn't be changed. Wdyt?
